components/lobste_rs/lobste_rs.app.mjs

```
@@ -1,11 +1,111 @@
+          let item = this.read();
+
+          while (item) {
+            for (const k in item) {
+              if (item[`rss:${k}`]) {
+                delete item[`rss:${k}`];
+                continue;
+              }
+              const o = item[k];
+              if (o == null || (typeof o === "object" && !Object.keys(o).length) || Array.isArray(o) && !o.length) {
+                delete item[k];
+                continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)